mitxgraders/formulagrader.py

```diff
@@ -107,6 +117,64 @@ def schema_config(self):
             Required('failable_evals', default=0): NonNegative(int)
         })
 
+    Utils = namedtuple('Utils', ['tolerance', 'within_tolerance'])
+
+    def get_comparer_utils(self):
+        """Get the utils for comparer function."""
+        def _within_tolerance(x, y):
+            return within_tolerance(x, y, self.config['tolerance'])
+        return self.Utils(tolerance=self.config['tolerance'],
+                          within_tolerance=_within_tolerance)
+
+    @staticmethod
+    def default_comparer(comparer_params, student_input, utils):
+        """
+        Default comparer function.
+
+        Assumes comparer_params is just the expected answer wrapped in a list.
+        """
+        return all([
+            utils.within_tolerance(comparer_param, student_input)
+            for comparer_param in comparer_params
+            ])
```

Comment:
I see some forward-thinking here (I suspect you're thinking forwards towards matrix/vector student input). However, I think this is getting ahead of ourselves for the moment, as we haven't defined how student_input works here, and having room for more than one right answer is confusing. Let's keep this simple and just go with `return utils.within_tolerance(comparer_params[0], student_input)`.